### PR TITLE
Setup repository

### PR DESCRIPTION
**Pull request by SanaTameem**
-Added HTML
-Added CSS
-Added Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <section2 id="#" class="main-section">
+              <h2><h2>tages of the projesct.</h2></h2>The Waterfall approach is a linear method with distinct goals
+              for each development phase. Teams that use waterfall spend a</section2 id="#">
+              <section3 id="#" class="main-section">
+                <h2><h2>tages of the projesct.</h2>
+              </h2>reasonable amount of time on product planning in the early
+              </section3> 
+
+              <section4 id="#"  class="main-section">
+                <h2>tages of the projesct.</h2> They create an extensive overview of the main goals and objectives and plan what the working process will look like. Waterfall teams strive to create detailed documentation before any of the engineering stages begin. Careful planning works well for projects with little to no changes in progress as it allows for precise budgeting and time estimates. However, waterfall planning has proven to be ineffective for long-term development as it doesn’t account for possible changes and contingencies on the go. According to KPMG Global Agile Survey, 81% of companies
```
Comment (SanaTameem) [inline index.html:26]: - [ ] It is better to avoid lots of indentations and spaces for having clean coding.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                  <h1>Agile and waterfall approaches</h1d>
+            <section1 id="#" class="main-section">
+              <h2>Agile and waterfall approaches</h2>The documentation types that the team produces and
+              its scope depending on the software development approach that was chosen. There are two main ones: agile and waterfall. Each is unique in terms of accompanying documentation.
+            </section1>
+
+            <section2 id="#" class="main-section">
+              <h2><h2>tages of the projesct.</h2></h2>The Waterfall approach is a linear method with distinct goals
+              for each development phase. Teams that use waterfall spend a</section2 id="#">
+              <section3 id="#" class="main-section">
```
Comment (SanaTameem) [inline index.html:20]: - [ ] Instead of using section1 , section2 etc...  it is better to use the section tag alone.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+main {
+  background-color: gray;
+}
+
+h1 {
+  color: orange;
+}
```
Comment (SanaTameem) [inline style.css:7]: - [ ] It is better to use hex codes or RGB values for colors instead of using the original names of them.

**Review by SanaTameem — COMMENTED** [review]
Some changes required.

**Review by Ibizugbe — APPROVED** [review]
Hi @SanaTameem 👋👋, 

ShakeAndBakeShakeNBakeGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Github inline comment feature has been used👍
- [x] Comments are clear and polite👍

## Optional suggestions

- [ ] [OPTIONAL] You can make your summary more descriptive by making use of the templates used by the code reviewers here at microverse😉 This will make your summary more descriptive and polite😎

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**